modules/ide-integration/src/main/resources/pga/airavata-php-gateway/app/controllers/ExperimentController.php

```diff
@@ -129,8 +129,42 @@ public function createSubmit()
     public function summary()
     {
         $experiment = null;
+        $experiment_id = null;
+        // set the experiment Id using the jobId
+        if(isset($_GET['jobId'])){
+            Log::debug("Searching experiment summary using jobId");
+            $limit = 1;
+            $pageNo = 0;
+            $inputs["search-key"] = "jobId";
+            $inputs["search-value"] = $_GET['jobId'];
+            try {
+                $expContainer = ExperimentUtilities::get_expsearch_results_with_pagination($inputs, $limit,
+                    $pageNo);
+            } catch (AuthorizationException $ae) {
+
+                Log::error("Experiment wasn't found", array("message" => $enf->getMessage(), "username" => Session::get("username"), "gateway_id" => Session::get("gateway_id")));
```

Comment:
Change the error message. This is an authorization error

Reply:
I have fixed it.